lib/counter_culture/helpers.rb

```diff
@@ -0,0 +1,138 @@
+module CounterCulture
+  module Helpers
```

Comment:
I'm not a fan of this name. I would probably call this `ActiveRecordExtensions` to make clear that this gets included into `ActiveRecord::Base`. `Helpers` makes it sound like it's just internally used when the opposite is the case.